awx/settings/defaults.py

```diff
@@ -73,6 +71,7 @@ def IS_TESTING(argv=None):
 # timezone as the operating system.
 # If running in a Windows environment this must be set to the same as your
 # system time zone.
+USE_TZ = False
```

Comment:
Note there's another `USE_TZ = True` below on line 89, which will override setting here.

Reply:
Yeah, this was accidentally added.